cl/search/tests/tests.py

```diff
@@ -910,6 +911,27 @@ def test_handle_unbalanced_parentheses(self) -> None:
         )
         self.assertNotIn("Did you mean", r.content.decode())
 
+    def test_round_estimated_search_counts(self) -> None:
+        """Confirm search counts above the threshold are properly rounded"""
+
+        tests = [
+            (13, 13),  # Below ELASTICSEARCH_CARDINALITY_PRECISION threshold
+            (109, 109),
+            (809, 809),
+            (1074, 1074),
+            (
+                11740,
+                11000,
+            ),  # Above ELASTICSEARCH_CARDINALITY_PRECISION threshold
+            (367740, 360000),
+            (7867740, 7800000),
+            (95367740, 95000000),
+            (436307740, 430000000),
+        ]
```

Comment:
I go pretty blind reading things like this. Perhaps:

```suggestion
        tests = [
            (13, 13),  # Below ELASTICSEARCH_CARDINALITY_PRECISION threshold
            (109, 109),
            (809, 809),
            (1_074, 1_074),
            (
                11_740,
                11_000,
            ),  # Above ELASTICSEARCH_CARDINALITY_PRECISION threshold
            (367_740, 360_000),
            (7_867_740, 7_800_000),
            (95_367_740, 95_000_000),
            (436_307_740, 430_000_000),
        ]
```

Reply:
Fixed.